other/triplet_sum.py

```diff
@@ -0,0 +1,87 @@
+"""
+Given an array of integers and another integer target,
+we are required to find a triplet from the array such that it's sum is equal to
+the target.
+"""
+from typing import List, Tuple
+from itertools import permutations
+from timeit import repeat
+from random import randint

```

Comment:
```suggestion
from itertools import permutations
from random import randint
from timeit import repeat
from typing import List, Tuple
```
Please keep import sorted to make it easy to spot duplicate entries.